src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a type of Task that
+ * needs to be done by a set date and time.
+ */
+public class Deadline extends Task {
+
+    protected LocalDateTime by;
+
+    /**
+     * Class constructor.
+     *
+     * @param description the details of the deadline.
+     * @param by          the date and time the deadline needs to be done by.
```

Comment:
There are spaces here in the javadocs. Could've been on accident but u should remove it to avoid the docs to look weird haha